tests/test_bgp_af.rb

```diff
@@ -106,6 +118,38 @@ def test_collection_not_empty
     end
   end
 
+  def config_ios_xr_dependencies(asn)
+    # These dependencies are required on ios xr
```

Comment:
Do we need to add any of this configuration to the BgpAf class itself? I know Rich had to add a few things to the Bgp class along these lines.